UnityProject/Assets/Scripts/Gateway/StationGateway.cs

```diff
@@ -189,31 +196,29 @@ public virtual void TransportObjectsItems(ObjectBehaviour objectsitems)
 
 	public virtual void SetOnline()
 	{
-		for (int i = 0; i < Sprites.Length; i++)
+		for (int i = Count; i <= Count + 8; i++)
 		{
-			Sprites[i].sprite = Online[i];
+			Sprites[i - Count].sprite = Online[i];
 		}
 
-		Centre.sprite = Online[Count + 8];
-		Count += 1;
+		Count += 9;
 
-		if (Count > 2)
+		if (Count > 26)
 		{
 			Count = 0;
 		}
 	}
 
 	public virtual void SetOffline()
 	{
-		for (int i = 0; i < Sprites.Length; i++)
+		for (int i = Count; i <= Count + 8; i++)
 		{
-			Sprites[i].sprite = Offline[i];
+			Sprites[i - Count].sprite = Offline[i];
 		}
 
-		Centre.sprite = Offline[Count + 8];
-		Count += 1;
+		Count += 9;
 
-		if (Count > 2)
+		if (Count > 26)
```

Comment:
Basically because this object has 9 different sprite renders, they all have to be changed, and each animation has 3 frames, 9*3 = 27 but because of 0 its 26.

Count is Counting the starting frame, so 0, 9, 18, ~27~ no frame at 27 so resets to 0